presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -663,6 +701,31 @@ public TimeStat getNextPageStats()
         return handler.process(body);
     }
 
+    private static PrestoException propagate(ResponseException exception)
```

Comment:
would be nice to extract this in a separate "extract .." commit

(i assume nothing material changed beyond that btw)